index.html

```diff
@@ -148,48 +165,70 @@ <h2 class="proj-title">Multi-Post Stories</h2>
           LET'S CONNECT
         </h2>
         <ul class="social-icons d-flex">
           <li>
-            <a href="https://twitter.com/lfmnovaes"><img src="icon/twitter.svg" alt="Twitter"/></a>
+            <a href="https://twitter.com/lfmnovaes"><img src="icon/twitter.svg"
+                alt="My Twitter" /></a>
           </li>
           <li>
-            <a href="https://www.linkedin.com/in/lfmnovaes/"><img src="icon/linkedin.svg" alt="Linkedin"/></a>
+            <a href="https://www.linkedin.com/in/lfmnovaes/"><img
+                src="icon/linkedin.svg" alt="My Linkedin" /></a>
           </li>
           <li>
-            <a href="https://medium.com/@lfmnovaes"><img src="icon/medium.svg" alt="Medium"/></a>
+            <a href="https://medium.com/@lfmnovaes"><img src="icon/medium.svg"
+                alt="My Medium" /></a>
           </li>
           <li>
-            <a href="https://github.com/lfmnovaes"><img src="icon/github.svg" alt="GitHub"/></a>
+            <a href="https://github.com/lfmnovaes"><img src="icon/github.svg"
+                alt="My GitHub" /></a>
           </li>
           <li>
-            <a href="#"><img src="icon/angellist.svg" alt="AngelList"/></a>
+            <a href="#"><img src="icon/angellist.svg" alt="My AngelList" /></a>
           </li>
         </ul>
         <button type="button" class="button 500">Get my resume</button>
       </div>
       <div class="about-ext d-flex">
         <ul class="about-items">
-          <li class="about-item d-flex font500">Languages<img src="icon/chevron-down.png" alt="chevron down"/></li>
-          <li class="lang-item d-flex font500"><img src="icon/javascript.png" class="pad10" alt="JavaScript"/><span class="pad10">JavaScript</span></li>
-          <li class="lang-item d-flex font500"><img src="icon/html.png" class="pad10" alt="HTML"/><span class="pad10">HTML</span></li>
-          <li class="lang-item d-flex font500"><img src="icon/css.png" class="pad10" alt="CSS"/><span class="pad10">CSS</span></li>
-          <li class="about-item d-flex border font500">Frameworks<img src="icon/chevron-right.png" alt="chevron right"/></li>
-          <li class="about-item d-flex border font500">Skills<img src="icon/chevron-right.png" alt="chevron right"/></li>
+          <li class="about-item d-flex font500" tabindex="0">Languages
+            <img src="icon/chevron-down.png" alt="chevron down"/></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/javascript.png" class="pad10" alt="JavaScript"/>
+            <span class="pad10">JavaScript</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/html.png" class="pad10" alt="HTML"/>
+            <span class="pad10">HTML</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/css.png" class="pad10" alt="CSS"/>
+            <span class="pad10">CSS</span></li>
```

Comment:
- [ ] Your use of alternative text for images is good practice for accessibility in case they do not/are yet to load. However, alternative text that is the same as nearby or adjacent text will be presented multiple times to screen readers or when images are unavailable. This would be redundant and inconvenient. Kindly replace these three image `alt` with empty/null alternative text (`alt=""`) because the content of the image is already provided in context through text.